docs/topics/multiplatform-mobile/multiplatform-mobile-dependencies.md

```diff
@@ -0,0 +1,97 @@
+[//]: # (title: Add dependencies to your project)
+
+<microformat>
+    <p>This is a part of the <strong>Getting started with Kotlin Multiplatform Mobile</strong> tutorial. Before proceeding, make sure you've completed previous steps:</p>
+    <p><img src="icon-1-done.svg" width="20" alt="First step"/> <a href="multiplatform-mobile-setup.md">Set up environment</a><br/><img src="icon-2-done.svg" width="20" alt="Second step"/> <a href="multiplatform-mobile-create-first-app.md">Create your first cross-platform app</a><br/><img src="icon-3.svg" width="20" alt="Third step"/> <strong>Add dependencies</strong><br/><img src="icon-4-todo.svg" width="20" alt="Fourth step"/> Upgrade your app<br/><img src="icon-5-todo.svg" width="20" alt="Fifth step"/> Wrap up your project</p>
+</microformat>
+
+You've already created your first cross-platform application, but one thing is still missing. It's hard to imagine
```

Comment:
I’d reword this intro to smth more neutral. (It’s hard to call it an actual application, and not one thing is missing, zillion things are missing ;))

Something like:

You've already created your first Kotlin Multiplatform Mobile project! Let’s now learn how to add dependencies to third-party libraries, which is necessary to build any successful cross-platform application.